client/lib/src/process_event.rs

```diff
@@ -19,38 +17,60 @@ pub fn process_event<UpdateServer>(
   view: &mut view::T,
   client: &client::T,
   event: Event,
+  key_repeated: &mut bool,
 ) where UpdateServer: FnMut(protocol::ClientToServer)
 {
   match event {
-    Event::KeyDown{keycode, repeat, ..} => {
-      keycode.map(|keycode| {
-        if !repeat {
-          key_press(update_server, view, client, keycode);
+    Event::WindowEvent { event: WindowEvent::CursorMoved { .. }, .. } => {}
+    Event::DeviceEvent { event: DeviceEvent::Motion { .. }, .. } => {}
+    Event::DeviceEvent { event: DeviceEvent::MouseMotion { .. }, .. } => {}
+    _ => println!("event: {:?}", event),
```

Comment:
Why are we printing everything else? I'm getting a lot of output from this.

Reply:
Just some debugging leftover. I can make it a `log!`.